src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+.header {
+  display: flex;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-left: 50px;
+  padding-right: 40px;
```

```suggestion
  padding: 0 50px;
```
Паддинги с обеих сторон по 50 px